src/defs.h

```diff
@@ -1203,6 +1208,8 @@ dumpiov_upto(struct tcb *, int len, kernel_ulong_t addr, kernel_ulong_t data_siz
 extern void
 dumpstr(struct tcb *, kernel_ulong_t addr, kernel_ulong_t len);
 
+extern uint64_t get_max_size(void);
```

Comment:
`get_max_size` is an implementation detail, please do not turn it into an interface.